config/overlays/odh/kustomization.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: opendatahub
+
+resources:
+  - ../../default
+
+patches:
+  # patch to remove default `system` namespace in ../../manager/manager.yaml
+  - path: delete-namespace.yaml
```

Comment:
We need additional patch here :
```
  - target:
      kind: ConfigMap
      name: llama-stack-k8s-operator-distribution-images
    patch: |-
      - op: replace
        path: /data
        value:
          rh-dev: quay.io/opendatahub/llama-stack:latest
```

@leseb Does the distribution value here looks good to you?

Reply:
Confirmed offline, and the naming `rh-dev` looks good